src/components/PersonaCard/Jquery.PersonaCard.js

```diff
@@ -8,50 +8,62 @@
  * @param  {jQuery Object}  One or more .ms-PersonaCard components
  * @return {jQuery Object}  The same components (allows for chaining)
  */
-(function ($) {
-  $.fn.PersonaCard = function () {
+(function ($) {    
+    /* Show/Hide the appropriate details for the selected Action */
+    function toggleDetails(target, $personaCard){		
+        if('undefined' === typeof target){
+            var targetIds = [];
+            $('.ms-PersonaCard-action.is-active').each(function(){
+                target = $(this).attr('data-detailsTargetName');
+		        if('undefined' !== typeof target){
+                    targetIds.push(target);
+                }
+            });
+            targetIds.forEach(function(a){toggleDetails(a, $personaCard);});
+        }
+        else{
```

Comment:
Coding convention thing, move this 'else' up a line, directly after the above closing curly brace.